Detectors/TRD/workflow/io/src/TRDRawTreeWriterSpec.cxx

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef STEER_DIGITIZERWORKFLOW_SRC_TRDDIGITWRITERSPEC_H_
+#define STEER_DIGITIZERWORKFLOW_SRC_TRDDIGITWRITERSPEC_H_
+
+#include "Framework/DataProcessorSpec.h"
+#include "DPLUtils/MakeRootTreeWriterSpec.h"
+#include "Framework/InputSpec.h"
+#include "DataFormatsTRD/Digit.h"
+#include "DataFormatsTRD/TriggerRecord.h"
+#include <SimulationDataFormat/ConstMCTruthContainer.h>
+#include <SimulationDataFormat/IOMCTruthContainerView.h>
+#include "TRDWorkflowIO/TRDRawTreeWriterSpec.h"
+
+namespace o2
+{
+namespace trd
+{
+
+template <typename T>
+using BranchDefinition = framework::MakeRootTreeWriterSpec::BranchDefinition<T>;
+
+o2::framework::DataProcessorSpec getTRDRawTreeWriterSpec(bool mctruth, bool writeTrigRec)
```

Comment:
Can it be that this class is not needed at all?

Reply:
Well yes, the idea was to write out an eventrecord per entry in the tree to speed things up, but I think it will be similarly quick if we split the digits files up, and parallel via dataframe.

Reply:
class removed